### PR TITLE
2715 Added padding to EmptyState and updated changelog, readme

### LINKED ISSUE #320: Empty State shall have padding at the root DOM
Same as https://github.com/pxblue/angular-component-library/issues/336

### PR DESCRIPTION
**Pull request by bkarambe**
Fixes #320 

#### Changes proposed in this Pull Request:
- Added a class content to EmptyState
- Added padding to content

#### Screenshots:
Screenshot 2022-01-12 at 12 16 24 AM

#### To Test:
- yarn start:storybook
- Open EmptyState and basic story
- Inspect content to see padding

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
 # Changelog
+## v5.3.4 (Not yet published)
+
+### Added
+
+-   Added class `content` to `<EmptyState>`.
+-   Added padding to `<EmptyState>`.
```
Comment (emclaug2) [inline CHANGELOG.md:7]: Please specify the padding; added `1rem` padding

Comment (emclaug2) [reply CHANGELOG.md:7]: Is there any reason for not being able to add the padding on the root class instead?

Comment (bkarambe) [reply CHANGELOG.md]: Root div element is taking height of window, so I added one more div which takes height of content and added flex.

**Comment by daileytj** [comment]
I'm stepping on your toes a bit @bkarambe, but I wanted to see if we could do this without adding another wrapper div. I opened a PR to do this without the wrapper div here: https://github.com/brightlayer-ui/react-component-library/pull/360

If that looks good we can close this PR and delete the branch.

**Comment by huayunh** [comment]
Question for @joebochill : do we want to link to issues from changelogs? Like this:
image

**Comment by joebochill** [comment]
> Question for @joebochill : do we want to link to issues from changelogs? Like this:
> image

Yes, I’m trying to steer us towards adopting that convention.

**Comment by bkarambe** [comment]
Closing this PR, as this is being already address in https://github.com/brightlayer-ui/react-component-library/pull/360

**Pull request closed without merging**